agents/lakeshore372/LS372_agent.py

```diff
@@ -175,6 +175,21 @@ def init_lakeshore_task(self, session, params=None):
         return True, 'Lakeshore module initialized.'
 
     def start_acq(self, session, params=None):
+        """acq(params=None)
+
+        Method to start data acquisition process.
+
+        The most recent data collected is stored in session.data in the
+        structure::
+
+            >>> session.data
+            {"fields":
+                {"Channel_01_T": 1.34,
+                 "Channel_01_R": 21234.02},
+             "timestamp": 1601919441.9610307}
```

Comment:
Can we make it so `session.data` contains the most recent data point taken for each channel? So like:
```
 {"fields": {
       "Channel_01": {"T": 1.34, "R": 21234.02, "timestamp": 1601919441.9610307},
       "Channel_02": {"T": 4.34, "R": 12345, "timestamp": 1601919480},
        ....
}}
```

Reply:
Those are identical, as they're sampled at the same time.

Reply:
Yeah, can do. This strays a bit from the actual field names as stored in the .g3 files, but I think makes the most sense.